pyproject.toml

```
@@ -54,6 +54,7 @@ dependencies = [
     "loguru==0.7.*",
     "flask==3.*",
     "tomlkit==0.13.*",
+    "textual==1.*",
```

```suggestion
    "textual==0.88.*",
```

That's kind of unexpected -- textual version numbers don't match up with the textual-dev version numbers.